scalar/README.md

```
@@ -0,0 +1,129 @@
+
+	// CacheAge defines the max-age for the Cache-Control header in seconds.
+	//
+	// Optional. Default: 0 (no cache)
+	CacheAge int
+}
+```
+
+### Default Config
+```go
+var ConfigDefault = Config{
+	Next:     nil,
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)